_posts/2023-06-21-data.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: I have large or sensible data. How should I proceed?
+date: 2023-06-21 00:00:00
+tags: reproducibility
+description: Describe how to handle large or sensible data files when submitting to Computo
+---
+
+## Large data sets
+
+If your submission materials contain files larger than 50MB, **especially data files**, they won’t fit on a git repository as is. For this reason, we encourage you to put your data or any materials you deem necessary on an external “open data” centered repository hub such a [Zenodo](https://zenodo.org/) or [OSF](https://osf.io/).
```

Comment:
I would add : or any long-term (emphasis on long-term) data repository that is standard in your scientific community or for a specific type of data/scientific application, and for which it is straight-forward to retrieve the data from a script code/notebook code (we highly encourage to use open platforms, ideally institutionally hosted).

Reply:
You are welcome to add your suggestion!